### PR TITLE
Omnibus

### PR DESCRIPTION
**Pull request by cznewt**

### CODE REVIEW

File: gitlab/server/service.sls [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{%- else %}
+- gitlab.server._service_src
+{%- endif %}
+
+{%- if server.initial_data is defined %}
+
+gitlab_backup_dirs:
+  file.directory:
+  - names:
+    - /root/gitlab/scripts
```
Comment (fpytloun) [inline gitlab/server/service.sls]: Don't abuse root's home directory for backups, etc. I would instead use /var/cache/salt/minion or /var/backups or some better location?

File: gitlab/server/service.sls [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - /root/gitlab/flags
+  - user: root
+  - group: root
+  - mode: 700
+  - makedirs: true
+
+/root/gitlab/scripts/restore.sh:
```
Comment (fpytloun) [inline gitlab/server/service.sls]: Same here, use /usr/local/bin or some better location

File: gitlab/meta/backupninja.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+backup:
+  gitlab:
+    fs_includes:
+    - /srv/leonardo/sites/{{ app_name }}/media
```
Comment (fpytloun) [inline gitlab/meta/backupninja.yml]: /srv/leonardo for gitlab?

File: gitlab/server/_service_pkg.sls [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{%- from "gitlab/map.jinja" import server with context %}
```
Comment (fpytloun) [inline gitlab/server/_service_pkg.sls:1]: It would be better to nest states here, eg.
```
service/init.sls
service/pkg.sls
service/source.sls
```

Comment (cznewt) [reply gitlab/server/_service_pkg.sls:1]: not 2day :)

**Pull request merged**